build/CoreGraphics/dll/CoreGraphics.def

```diff
@@ -242,6 +242,10 @@ LIBRARY CoreGraphics
         _CGContextCreateWithD2DRenderTarget
         _CGGetD2DFactory
         _CGGetPixelFormatProperties
+        _CGContextPushBeginDraw
+        _CGContextPopEndDraw
+        _CGContextEscapeBeginEndDrawStack
```

Comment:
These names are not ideal - they are vague (what does escaping the stack mean), and reveal the implementation detail (there is a stack).  The later is not a big deal, but essentially what you doing here is ability to deal with a layer, so it might make sense to just call this _CGContextForceEndDraw() and _CGContextForceBeginDraw(), or _CGContextBeginLayerDraw/EndLayerDraw().